cmd/cloud-controller-manager/main.go

```diff
@@ -82,6 +82,13 @@ func main() {
 		Constructor: nodeIpamController.startNodeIpamControllerWrapper,
 	}
 
+	controllerInitializers[names.ServiceLBController] = app.ControllerInitFuncConstructor{
+		InitContext: app.ControllerInitContext{
+			ClientName: "service-controller",
+		},
+		Constructor: startGkeServiceControllerWrapper,
+	}
```

Comment:
this is a new controller that has to be opt-in, not overriding the standard one and disabled by default, think this component is used by other OSS projects that do not necessarily use GKE.